samples/githubDispatch/command.js

```diff
@@ -0,0 +1,51 @@
+const { Octokit } = require('octokit')
+// argument format
+// /fusebot <user/repo> <workflow_id> <branch/tag name>
+
+// please update this with your own personal access token
+const personalAccessToken = "xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx"
```

Comment:
How about we ask people in README.md to specify `KEY={personal-access-token}` in the *Configuration* section of the editor, and then we use it as `ctx.configuation.KEY` in code? We don't want to promote folks storing secrets in code directly. Also, give people a hint in README.md how to obtain the personal access token (a link? a section in the repo?)

Reply:
Also, how about you return a usable exception back to slack/discord if folks actually do not set up that configuration property?

Reply:
Done